### PR TITLE
Login Auth, Calendar View

### PR DESCRIPTION
**Pull request by johnsington**
- added cookie to establish user session
- created calendar view
- created login page

### CODE REVIEW

File: app/scripts/FBUtil.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        if (res.status == 'connected'){
+          FB.api('/me', {fields: 'name,email'}, function(response) {
+            console.log(response);
+          });
+        }
+        else if (res.status == 'not_authorized') {
+          console.log('not authorized');
+        } else {
+          console.log(res); 
+
```
Comment (luisasanmartin) [inline app/scripts/FBUtil.js]: whitespace

File: app/scripts/FBUtil.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    }
+      
+    this.login = function() {
+      FB.login(function(res){
+        if (res.status == 'connected') {
+          FB.api('/me', {fields: 'name,email'}, function(response) {
+            response.fbToken = res.authResponse.accessToken;
+            api.login(response);
+            cookie.set('response-name', response.name, { expires: res.expiresIn});
+             window.location.assign('http://localhost:9000/');
```
Comment (luisasanmartin) [inline app/scripts/FBUtil.js]: spacing

File: app/scripts/FBUtil.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+          });
+        }
+        else {
+          console.log('An error occurred in GET /me');
```
Comment (luisasanmartin) [inline app/scripts/FBUtil.js:60]: todo for this

File: app/scripts/apiUtil.js [hunk 4]
```diff
@@ -2,7 +2,7 @@ import $ from 'jquery';
 import request from 'request';
 
 export default function () {
-	const url = 'https://sheltered-escarpment-22682.herokuapp.com/';
+	const url = 'https://sheltered-escarpment-22682.herokuapp.com';
```
Comment (luisasanmartin) [inline app/scripts/apiUtil.js:5]: this changes. dunno if its my bad or if it should be a config var.

**Review by luisasanmartin — APPROVED** [review]
Remove DS_Store

File: app/scripts/apiUtil.js (hunk 4 above)
Comment (johnsington) [reply app/scripts/apiUtil.js:5]: I'll set it as a config variable once we deploy the client to a heroku server.

**Pull request merged**